scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -382,20 +385,35 @@ <h3 class="header">Institute Settings</h3>
             </td>
             <td><!-- CADD score -->
               {% if variant.cadd_score %}
-                <span class="badge bg-secondary">{{ variant.cadd_score }}</span>
+                <span class="badge bg-secondary">{{ variant.cadd_score|round(2) }}</span>
               {% else %}
                 -
               {% endif %}
             </td>
             <td><!-- Rank score -->
-              <a href="#"><span class="badge rounded-pill bg-secondary">
+              <span class="badge rounded-pill bg-secondary">
               {{ variant.rank_score }}
               {% if variant.category == 'sv' %}
                 (v. {{case.sv_rank_model_version or 'na'}})
               {% else %}
                 (v. {{case.rank_model_version or 'na'}})
               {% endif %}
-              </span></a>
+              </span>
+            </td>
+            <!-- Revel scores -->
+            {% for score in [variant.revel, variant.revel_score] %}
+            <td>
+              {% if score %}
+                <span class="badge rounded-pill bg-secondary">
+                  {{score}}
```

Comment:
And we probably want to round those as well! 😀 
![Screenshot 2024-01-10 at 11 25 45](https://github.com/Clinical-Genomics/scout/assets/758570/1086dbd4-4be6-4dc5-9a8d-2cbcaa471419)

Reply:
Done